Documentation/production-users.md

```diff
@@ -0,0 +1,12 @@
+# Production users
+
+This document tracks people and use cases for rkt in production. [Join the community](https://github.com/coreos/rkt/), and help us keep the list up-to-date.
+
+## [BlaBlaCar](https://www.blablacar.com/)
+BlaBlaCar is a trusted car-pooling service based in France. They've [blogged about](http://blablatech.com/blog/why-and-how-blablacar-went-full-containers) the stability of rkt being a big appeal.
+
+## [Kumul.us](https://kumul.us/)
+Kumulus Technologies offers classes and services to optimize your Cloud. They've [blogged about](https://kumul.us/docker-youve-failed-me-again-rkt-please-save-me/) rkt's pod native features and Kubernetes support being a reason for their adoption.
+
+## [Kinvolk.io](https://kinvolk.io/blog/2016/02/welcome-rkt-1.0/)
```

Comment:
Perhaps this link should be switched to the naked domain, and a link to a real usecase (eg. https://kinvolk.io/blog/2017/02/using-custom-rkt-stage1-images-to-test-against-various-kernel-versions/) added in the long description.